_inc/client/components/product-card/index.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+import analytics from 'lib/analytics';
```

Comment:
Would be great to keep those imports in alphabetical order, like in Calypso. There are exceptions, like `React` and `PropTypes` are usually kept at the top.

Reply:
I am alphabetizing them by their imports, e.g. `components/card`, `components/gridicon`, `lib/analytics` instead of `analytics`, `Card`, `Gridicon`. I did this because with the latter method it wasn't clear to me how to sort multiple imports like `import { ItemOne, ItemTwo } from '...'`. I'm happy to sort any way though as long as there is a consistent rule/pattern :)